frontend/src/lib/query/createTableColumnsFromQuery.ts

```diff
@@ -215,10 +215,12 @@ const getDynamicColumns: GetDynamicColumns = (queryTableData, query) => {
 			}
 
 			series.forEach((seria) => {
-				Object.keys(seria.labels).forEach((label) => {
-					if (label === currentQuery?.queryName) return;
+				seria.labelsArray?.forEach((lab) => {
```

Comment:
seria ?

Reply:
used the same itself , already present code